packages/service-library/tests/aiohttp/test_docker_utils.py

```diff
@@ -0,0 +1,120 @@
+# pylint: disable=protected-access
+from collections.abc import Awaitable, Callable
+from typing import Any
+from unittest.mock import call
+
+import pytest
+from models_library.docker import DockerGenericTag
+from pydantic import parse_obj_as
+from pytest_mock import MockerFixture
+from servicelib import progress_bar
+from servicelib.aiohttp.docker_utils import retrieve_image_layer_information
+from servicelib.docker_utils import pull_image
+from settings_library.docker_registry import RegistrySettings
+
+
+@pytest.mark.parametrize(
+    "service_repo, service_tag",
+    [
+        ("itisfoundation/sleeper", "1.0.0"),
+        ("itisfoundation/sleeper", "2.2.0"),
+        (
+            "itisfoundation/sleeper",
+            "sha256:a6d9886311721d8d341068361ecf9998a3c7ecb0efb23ebac553602c2eca1f8f",
+        ),
+    ],
+)
+async def test_retrieve_image_layer_information(
+    remove_images_from_host: Callable[[list[str]], Awaitable[None]],
+    registry_settings: RegistrySettings,
+    osparc_service: dict[str, Any],
+    service_repo: str,
+    service_tag: str,
+):
+    # clean first
+    image_name = f"{service_repo}:{service_tag}"
+    if "sha256" in service_tag:
+        image_name = f"{service_repo}@{service_tag}"
+    await remove_images_from_host([image_name])
```

Comment:
why not testing that they are actually removed? e.g. checking that they are not listed

Reply:
done